data/scripts.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const {PRNG} = require('./../prng');
```

Comment:
I did *actually* catch this but must have screwed up the rebase, apologies! Did you fix this yourself?

Also - how did CI pass?